crates/libp2p-networking/src/network/def.rs

```diff
@@ -144,25 +143,13 @@ impl NetworkDef {
 /// Request/response functions
 impl NetworkDef {
     /// Add a direct request for a given peer
-    pub fn add_direct_request(&mut self, peer_id: PeerId, data: Vec<u8>, retry_count: u8) {
-        let request = DMRequest {
-            peer_id,
-            data,
-            backoff: ExponentialBackoff::default(),
-            retry_count,
-        };
-        self.direct_message.add_direct_request(request);
+    pub fn add_direct_request(&mut self, peer_id: PeerId, data: Vec<u8>) -> OutboundRequestId {
+        self.direct_message.send_request(&peer_id, data)
     }
 
     /// Add a direct response for a channel
     pub fn add_direct_response(&mut self, chan: ResponseChannel<Vec<u8>>, msg: Vec<u8>) {
-        self.direct_message.add_direct_response(chan, msg);
-    }
-}
-
-impl From<DMEvent> for NetworkEventInternal {
-    fn from(event: DMEvent) -> Self {
-        Self::DMEvent(event)
+        let _ = self.direct_message.send_response(chan, msg);
```

Comment:
What is the `let _` being used for here?

Reply:
It's suppressing an error (and clipply).  I can log the error, but this is from the other PR

Reply:
tbh I'm not sure why this shows up even though this PR is to merge into the other branch